src/packages/client/src/generation/generateClient.ts

```diff
@@ -246,9 +246,14 @@ export async function generateClient({
     )
   }
   if (transpile) {
-    for (const filePath of Object.values(enginePath)) {
+    if (process.env.NETLIFY) await makeDir('/tmp/prisma-engines')
```

Comment:
I'm surprised prettier/linting didn't do anything, we usually prefer 
```suggestion
    if (process.env.NETLIFY) {
    	await makeDir('/tmp/prisma-engines')
    }
```